### PR TITLE
[v8] Update Modal SDK reference, add new adapters, and update common web

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

File: docs/sdk/pnp/web/adapters/default-solana-adpater.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    chainId: "0x3",
+    displayName: "Solana Testnet",
+    chainNamespace: CHAIN_NAMESPACES.SOLANA,
+    tickerName: "SOLANA",
+    ticker: "SOL",
+    decimals: 18,
+    rpcTarget: "https://api.testnet.solana.com",
+    blockExplorerUrl: "https://explorer.solana.com/?cluster=testnet",
+    logo: "https://images.toruswallet.io/sol.svg"
+};
```
Comment (rtomas) [inline docs/sdk/pnp/web/adapters/default-solana-adpater.mdx:82]: you can add "isTestnet:true"

File: docs/sdk/pnp/web/modal/initialize.mdx [hunk 2]
```diff
@@ -57,6 +57,45 @@ This Web3Auth constructor takes an object with `Web3AuthOptions` as input.
...
+  logo: "https://images.toruswallet.io/eth.svg",
+};
+
+const ethereumPrivateKeyProvider = EthereumPrivateKeyProvider({
+  config: {chainConfig}
+});
+
+const web3auth = new Web3Auth({
+  clientId: "", // Get your Client ID from the Web3Auth Dashboard
+  web3AuthNetwork: "sapphire_mainnet",
```
Comment (rtomas) [inline docs/sdk/pnp/web/modal/initialize.mdx:89]: you can change text "sapphire_mainnet" to type web3AuthNetwork: 

import { WEB3AUTH_NETWORK } from "@web3auth/base";

WEB3AUTH_NETWORK.SAPPHIRE_MAINNET

File: src/common/sdk/pnp/web/_add-chain.mdx [hunk 3]
```diff
@@ -60,6 +60,10 @@ export type CustomChainConfig = {
    * Number of decimals for the currency ticker (e.g: 18)
    */
   decimals?: number;
+   /**
+   * Logo for the chain
+   */
+  logo: string;
 };
```
Comment (rtomas) [inline src/common/sdk/pnp/web/_add-chain.mdx:67]: "isTestnet" is missing in " type CustomChainConfig.
type CustomChainConfig = {"
/**
    Whether the network is testnet or not*/
    isTestnet?: boolean;

File: docs/sdk/pnp/web/adapters/default-solana-adpater.mdx (hunk 1 above)
Comment (AyushBherwani1998) [reply docs/sdk/pnp/web/adapters/default-solana-adpater.mdx:82]: Done

File: docs/sdk/pnp/web/modal/initialize.mdx (hunk 2 above)
Comment (AyushBherwani1998) [reply docs/sdk/pnp/web/modal/initialize.mdx:89]: Done

**Pull request merged**